universal-application-tool-0.0.1/test/views/questiontypes/TextQuestionRendererTest.java

```diff
@@ -0,0 +1,62 @@
+package views.questiontypes;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.google.common.collect.ImmutableMap;
+import j2html.tags.Tag;
+import java.util.Locale;
+import org.junit.Before;
+import org.junit.Test;
+import repository.WithPostgresContainer;
+import services.Path;
+import services.applicant.ApplicantData;
+import services.applicant.ApplicantQuestion;
+import services.question.TextQuestionDefinition;
+
+public class TextQuestionRendererTest extends WithPostgresContainer {
+  private static final TextQuestionDefinition textQuestionDefinition =
```

Comment:
super nit: Java convention is for constant names to be upper snake case: TEXT_QUESTION_DEFINITION

Reply:
done!